config/passport.js

```diff
@@ -0,0 +1,84 @@
+var GoogleStrategy = require('passport-google-oauth').OAuth2Strategy;
+var LocalStrategy = require('passport-local').Strategy;
+var BearerStrategy = require('passport-http-bearer').Strategy;
+var configAuth = require('./settings');
+var connection = require('./database');
+
+module.exports = function(passport) {
+
+	passport.serializeUser(function(user, done){
+		done(null, user);
+	});
+
+	passport.deserializeUser(function(user, done){
+		done(null, user);
+	});
+
+	passport.use(new GoogleStrategy({
+	    clientID: configAuth.googleAuth.clientID,
+	    clientSecret: configAuth.googleAuth.clientSecret,
+	    callbackURL: configAuth.googleAuth.callbackURL,
+	    access_type: configAuth.googleAuth.access_type
+	  },
+	  function(req, accessToken, refreshToken, profile, done) {
+	  	process.nextTick(function(){
+		  	profileData = {email: profile.emails[0].value, token: accessToken, google_id: profile.id, name: profile.displayName};
+		  	connection.query('SELECT * from google_user_accounts where email=?', profileData.email, function (err, rows, fields){
```

Comment:
Hi @yukie1412 I have made changes to incorporate orm based queries using sequelize. It works fine. Its better to switch to orm based queries as early as possible, migrating it later might cause problems to some functionalities and a lot of time will be spent on regression/sanity tests. I shall make the commits if you approve this approach else I'll work on developing the orm like library.